mender-convert.sh

```diff
@@ -5,28 +5,36 @@ cat << EOF
 
 Mender conversion tool
 
-A tool for taking an existing embedded image (Debian, Ubuntu, Raspbian, etc) and
-convert it to a Mender image by restructuring partition table and adding
+A tool for taking an existing embedded image (Debian, Ubuntu, Raspbian, etc)
+and convert it to a Mender image by restructuring partition table and adding
 necessary files.
 
-Usage: $0 [prepare_image | make_sdimg | install_mender | install_bootloader | make_all] [options]
+Usage: $0 [raw-disk-image-shrink-rootfs |
+           raw-disk-image-create-partitions |
+           install-mender-to-mender-disk-image |
+           install-bootloader-to-mender-disk-image |
+           from-raw-disk-image] [options]
 
 Actions:
```

Comment:
If incorporating above, would uppercase "ACTIONS:"

Reply:
Here instead of "COMMANDS" I would use "General commands:"

Reply:
:+1: